plugins/main/public/components/common/welcome/components/vuls_panel/vuls_welcome_panel.tsx

```diff
@@ -0,0 +1,204 @@
+import {
+  EuiPanel,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+  EuiToolTip,
+  EuiButtonIcon,
+  EuiSpacer,
+  EuiLink,
```

Comment:
This component is not being used in the codebase, so it can be removed.

Reply:
Solved in https://github.com/wazuh/wazuh-dashboard-plugins/pull/7085/commits/fc6f1779f0b53f7fb1018d1499af7137c8819f36